### ISSUE TITLE
[OpenMW] Handle groundcover plugins

### ISSUE THREAD

**Issue by poperigby** [limo-app/limo#91]
Groundcover plugins have a special entry in `openmw.cfg` like this:

```ini
groundcover=example.esp
```

The can't be enabled as regular plugins, or they won't work correctly.

**Comment by limo-app** [comment]
Since i do not play OpenMW myself, i lack experience with these plugin types. Am i right in assuming that there is no way to auto detect whether a plugin is a groundcover plugin using LOOT? If so, this would probably have to be handled by offering users the option to tag plugins as groundcover.

**Comment by poperigby** [comment]
Yeah, there's no way to tell whether a plugin is supposed to be used for groundcover. It's kind of a nasty situation, but there are [plans](https://gitlab.com/OpenMW/openmw/-/issues/6263) to do away with groundcover plugins in the future and just use mesh flags.

For now, an option to flag them would be helpful.

**Comment by limo-app** [comment]
7e184ac9fe79144eea5d6950b623cb22dcc7b261 added an option to flag plugins as groundcover, among other changes. See my post in https://github.com/limo-app/limo/issues/95#issuecomment-2600983864
